src/elements/FormTextInput/FormTextInput.tsx

```diff
@@ -22,6 +23,7 @@ interface FormTextInputProps {
 
 export default ({
 	text,
+	placeholder,
```

Comment:
`placeholder = '',` will set the default value of placeholder to an empty string. Now you can delete every instance where you passed an empty string placeholder into the component